server/commands/subscriptionCreator.ts

```diff
@@ -36,8 +36,23 @@ export default async function subscriptionCreator({
     paranoid: false,
   });
 
-  // Don't emit an event if a new subscription
-  // wasn't created.
+  // Fetched an already deleted subscription.
+  if (subscription.deletedAt) {
+    subscription.update({ deletedAt: null });
```

Comment:
```suggestion
    subscription.update({ deletedAt: null }, { transaction });
```

Should be updated in the same transaction